src/helpers/check-merge-safety.ts

```diff
@@ -79,13 +82,14 @@ const getMergeSafetyMessage = async (
   });
   const fileNamesWhichBranchIsBehindOn = filesWhichBranchIsBehindOn?.map(file => file.filename) ?? [];
 
-  const shouldOverrideSafetyCheck = override_filter_globs
-    ? micromatch(fileNamesWhichBranchIsBehindOn, override_filter_globs.split('\n')).length > 0
+  const globalFilesOutdatedOnBranch = override_filter_globs
+    ? micromatch(fileNamesWhichBranchIsBehindOn, override_filter_globs.split('\n'))
     : override_filter_paths
-    ? fileNamesWhichBranchIsBehindOn.some(changedFile => override_filter_paths.split(/[\n,]/).includes(changedFile))
-    : false;
+    ? fileNamesWhichBranchIsBehindOn.filter(changedFile => override_filter_paths.split(/[\n,]/).includes(changedFile))
+    : [];
 
-  if (shouldOverrideSafetyCheck) {
+  if (globalFilesOutdatedOnBranch.length) {
+    core.error(buildErrorMessage(globalFilesOutdatedOnBranch));
```

Comment:
throwing `the following projects are outdated` might be misleading for filenames like `package.json`

Reply:
oh crap good catch, that was not intended!